### PR TITLE
Remove tfm as package prefix for EL8

### PR DESCRIPTION
**Pull request by mjivraja**
We need to remove the `tfm` as a package prefix as it no longer supports EL8 in Project version 3.4 and later.

https://bugzilla.redhat.com/show_bug.cgi?id=2138391

* [X] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [X] Foreman 3.5/Katello 4.7
* [X] Foreman 3.4/Katello 4.6
* [X] Foreman 3.3/Katello 4.5
* [ ] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/common/modules/proc_installing-webhooks-plugin.adoc [hunk 1]
```diff
@@ -15,5 +15,5 @@ After installing webhooks, you can configure {ProjectServer} to send webhook req
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# yum install tfm-rubygem-hammer_cli_foreman_webhooks
+# yum install rubygem-hammer_cli_foreman_webhooks
```
Comment (ekohl) [inline guides/common/modules/proc_installing-webhooks-plugin.adoc:18]: A better way is to use `--enable-foreman-cli-plugin-webhooks` as an installer argument. It can be added to the previous command so only a single installer run is needed. Note that also works on EL7 and Debian/Ubuntu.

**Review by ekohl — CHANGES_REQUESTED** [review]
I know I'm enlarging the scope, but I consider these to be bugs that we've had for a long time.

It may be a good idea to split the work into multiple PRs (like one for discovery, one for scc_manager, one for webhooks and one for azure), but I'll leave that up to you.

File: guides/common/modules/proc_installing-the-scc-manager.adoc [hunk 2]
```diff
@@ -14,7 +14,7 @@ Perform the following steps to install the _SCC Manager_ plug-in on your {Projec
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {package-install-project} -y tfm-rubygem-foreman_scc_manager
+# {package-install-project} -y rubygem-foreman_scc_manager
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/modules/proc_installing-the-scc-manager.adoc:17]: Thanks for letting me know, I'll work on this in a separate PR some day!

File: guides/common/modules/proc_installing-the-discovery-service.adoc [hunk 3]
```diff
@@ -9,7 +9,7 @@ Complete the following procedure to enable the Discovery service on {SmartProxyS
 ifdef::satellite,orcharhino[]
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {package-install-project} foreman-discovery-image tfm-rubygem-smart_proxy_discovery
+# {package-install-project} foreman-discovery-image rubygem-smart_proxy_discovery
```
[the remark this answers is not in the document]
Comment (mjivraja) [reply guides/common/modules/proc_installing-the-discovery-service.adoc]: Addressed :+1:

File: guides/common/modules/proc_installing-webhooks-plugin.adoc (hunk 1 above)
Comment (mjivraja) [reply guides/common/modules/proc_installing-webhooks-plugin.adoc]: Addressed :+1:

File: guides/common/modules/proc_uninstalling-microsoft-azure-plugin.adoc [hunk 4]
```diff
@@ -8,7 +8,7 @@ If you have previously installed the Microsoft Azure plugin but don't use it any
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# yum remove -y tfm-rubygem-foreman_azure_rm tfm-rubygem-ms_rest_azure
+# yum remove -y rubygem-foreman_azure_rm rubygem-ms_rest_azure
```
[the remark this answers is not in the document]
Comment (mjivraja) [reply guides/common/modules/proc_uninstalling-microsoft-azure-plugin.adoc]: Noted Ewoud. Many of the files in the repo are still unexplored from my side. But, I will be careful from now onwards on this observation. :+1:

File: guides/common/modules/con_configuring-the-discovery-service.adoc [hunk 5]
```diff
@@ -39,13 +39,13 @@ endif::[]
 
 To use {ProjectServer} to provide the Discovery image, install the following RPM packages:
```
[the remark this answers is not in the document]
Comment (mjivraja) [reply guides/common/modules/con_configuring-the-discovery-service.adoc]: Addressed :+1:

File: guides/common/modules/proc_uninstalling-microsoft-azure-plugin.adoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (ekohl) [reply guides/common/modules/proc_uninstalling-microsoft-azure-plugin.adoc]: Yes, I'd say that ideally speaking we'd go through every file and review it. There are many things that we can do better. Right now I often address things ad-hoc or when I read through them, but a concentrated effort would be good.

File: guides/common/modules/proc_configuring-the-discovery-service.adoc [hunk 6]
```diff
@@ -37,27 +37,37 @@ endif::[]
...
+----
+# {foreman-installer} \
+--enable-foreman-plugin-discovery \
+--enable-foreman-proxy-plugin-discovery \
+--foreman-proxy-plugin-discovery-install-images=false
+----
 endif::[]
-
-The `tfm-rubygem-smart_proxy_discovery` package configures {SmartProxyServer}, such as the integrated {SmartProxy} of {ProjectServer}, to act as a proxy for the Discovery service.
+. Install `foreman-discovery-image`:
```
Comment (ekohl) [inline guides/common/modules/proc_configuring-the-discovery-service.adoc:62]: We only have `foreman-discovery-image` in satellite, there is no upstream equivalent. I did see previously there it was also used for `orcharhino`. @maximiliankolb is this also something that Orcharhino duplicated?

Please move this into the `ifdef::satellite` block.

File: guides/common/modules/proc_configuring-the-discovery-service.adoc [hunk 7]
```diff
@@ -37,27 +37,37 @@ endif::[]
 hosts or virtual machines can be created with incorrect network configurations. On Dell servers, the new naming scheme can be disabled via `biosdevname=1` kernel command line option, for other hardware or virtual machines, the new naming scheme can be completely turned off via `net.ifnames=1`.
 endif::[]
 
-To use {ProjectServer} to provide the Discovery image, install the following RPM packages:
-
-* `tfm-rubygem-foreman_discovery`
-ifdef::satellite[]
-* `foreman-discovery-image`
+.Procedure
+. To use {ProjectServer} to provide the Discovery image, run the following commands:
```
Comment (ekohl) [inline guides/common/modules/proc_configuring-the-discovery-service.adoc:41]: It only lists a single installer command, so singular:
```suggestion
. To use {ProjectServer} to provide the Discovery image, run the following command:
```

I'd also say that it does more than provide the discovery image. It installs the Foreman Discovery plugin, which is required if you want to use the Foreman Proxy Discovery plugin from some Foreman Proxy. Then it also installs the Foreman Proxy Discovery Plugin, which technically you don't need on `{ProjectServer}` if you have separate Foreman Proxies, but our instructions typically say that because we don't assume separate Foreman Proxies.

So I'd rephrase it to:
```suggestion
. Install Discovery on {ProjectServer}:
```

However, feel free to come up with some alternative here.

File: guides/common/modules/proc_installing-the-discovery-service.adoc [hunk 8]
```diff
@@ -21,21 +23,15 @@ ifdef::foreman-el,katello[]
 --foreman-proxy-plugin-discovery-source-url=http://downloads.theforeman.org/discovery/releases/3.5/
 ----
 endif::[]
-. Restart {Project} services:
-+
+. Install `foreman-discovery-image`:
```
Comment (ekohl) [inline guides/common/modules/proc_installing-the-discovery-service.adoc:26]: Like above, this is limited to satellite (and possibly orcharhino).

File: guides/common/modules/proc_installing-webhooks-plugin.adoc [hunk 9]
```diff
@@ -9,11 +9,5 @@ After installing webhooks, you can configure {ProjectServer} to send webhook req
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {foreman-installer} --enable-foreman-plugin-webhooks
```
Comment (ekohl) [inline guides/common/modules/proc_installing-webhooks-plugin.adoc:12]: This must remain. It was only that `{foreman-installer} --enable-foreman-cli-plugin-webhooks` is a proper replacement for `yum install tfm-rubygem-hammer_cli_foreman_webhooks`.

**Review by ekohl — CHANGES_REQUESTED** [review]

File: guides/common/modules/proc_installing-webhooks-plugin.adoc (hunk 9 above)
Comment (mjivraja) [reply guides/common/modules/proc_installing-webhooks-plugin.adoc]: Addressed :+1:

File: guides/common/modules/proc_installing-the-discovery-service.adoc (hunk 8 above)
Comment (mjivraja) [reply guides/common/modules/proc_installing-the-discovery-service.adoc:27]: From the earlier lines, I have considered including `Orcharhino` alongside. Addressed :+1:

File: guides/common/modules/proc_configuring-the-discovery-service.adoc (hunk 7 above)
Comment (mjivraja) [reply guides/common/modules/proc_configuring-the-discovery-service.adoc]: Addressed :+1:

File: guides/common/modules/proc_configuring-the-discovery-service.adoc (hunk 6 above)
Comment (mjivraja) [reply guides/common/modules/proc_configuring-the-discovery-service.adoc:61]: I have considered `Orcharhino` from `guides/common/modules/proc_installing-the-discovery-service.adoc`

Correct me if I am wrong. Addressed :+1:

File: guides/common/modules/proc_configuring-the-discovery-service.adoc [hunk 10]
```diff
@@ -37,27 +37,36 @@ endif::[]
...
 
-To use {ProjectServer} to provide the Discovery image, install the following RPM packages:
-
-* `tfm-rubygem-foreman_discovery`
-ifdef::satellite[]
-* `foreman-discovery-image`
+.Procedure
+. Install Discovery on {ProjectServer}:
++
+ifdef::foreman-el,katello[]
```
Comment (ekohl) [inline guides/common/modules/proc_configuring-the-discovery-service.adoc:43]: Can we keep it symmetrical?
```suggestion
ifndef::satellite,orcharhino[]
```

File: guides/common/modules/proc_installing-the-discovery-service.adoc [hunk 11]
```diff
@@ -9,7 +9,9 @@ Complete the following procedure to enable the Discovery service on {SmartProxyS
 ifdef::satellite,orcharhino[]
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {package-install-project} foreman-discovery-image tfm-rubygem-smart_proxy_discovery
+# {foreman-installer} \
+--enable-foreman-proxy-plugin-discovery \
+--foreman-proxy-plugin-discovery-install-images=false
 ----
 endif::[]
 ifdef::foreman-el,katello[]
```
Comment (ekohl) [inline guides/common/modules/proc_installing-the-discovery-service.adoc:17]: While you're at it, mind changing this too?
```suggestion
ifndef::satellite,orcharhino[]
```

File: guides/common/modules/proc_configuring-the-discovery-service.adoc [hunk 12]
```diff
@@ -37,27 +37,36 @@ endif::[]
...
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} \
+--enable-foreman-plugin-discovery \
+--enable-foreman-proxy-plugin-discovery \
+--foreman-proxy-plugin-discovery-install-images=true \
+--foreman-proxy-plugin-discovery-source-url=http://downloads.theforeman.org/discovery/releases/3.5/
```
Comment (ekohl) [inline guides/common/modules/proc_configuring-the-discovery-service.adoc:50]: @evgeni I'd like to get your thoughts on this. The specific URL is copied from the existing documentation. If we compare it to the default `latest` (https://github.com/theforeman/puppet-foreman_proxy/blob/6bd3c34a0771bd87b6640360ad8f216b0b285d77/manifests/plugin/discovery/params.pp#L6) I wonder what's better. Using `latest` could be problematic for older releases, but on the other hand, we only support the last 2 releases and I don't really expect compatibility issues. Additionally, I don't expect users to change this on upgrade so should we recommend to omit this option and rely on the default?

File: guides/common/modules/proc_installing-the-scc-manager.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (ekohl) [reply guides/common/modules/proc_installing-the-scc-manager.adoc:17]: I think it should be this: https://github.com/theforeman/foreman-documentation/pull/1800

File: guides/common/modules/proc_configuring-the-discovery-service.adoc (hunk 12 above)
Comment (ekohl) [reply guides/common/modules/proc_configuring-the-discovery-service.adoc:50]: @mjivraja then please drop this line and `--foreman-proxy-plugin-discovery-install-images=true` (since that's a default too). Then remove that too in `guides/common/modules/proc_installing-the-discovery-service.adoc` to keep it consistent.

Comment (evgeni) [reply guides/common/modules/proc_configuring-the-discovery-service.adoc:50]: install images is false by default:
https://github.com/theforeman/puppet-foreman_proxy/blob/6bd3c34a0771bd87b6640360ad8f216b0b285d77/manifests/plugin/discovery/params.pp#L4

Comment (ekohl) [reply guides/common/modules/proc_configuring-the-discovery-service.adoc:50]: Oh right, thanks for checking.

File: guides/common/modules/proc_installing-the-discovery-service.adoc (hunk 11 above)
Comment (mjivraja) [reply guides/common/modules/proc_installing-the-discovery-service.adoc]: I never mind at all, Ewoud :heart: 

Addressed :+1:

File: guides/common/modules/proc_configuring-the-discovery-service.adoc (hunk 10 above)
Comment (mjivraja) [reply guides/common/modules/proc_configuring-the-discovery-service.adoc]: Addressed :+1:

**Comment by ekohl** [comment]
@mjivraja would you mind rebasing and squashing your commits? That should also fix the build issue.

**Comment by mjivraja** [comment]
> @mjivraja would you mind rebasing and squashing your commits? That should also fix the build issue.

Agree. Addressed :+1:

File: guides/common/modules/proc_configuring-the-discovery-service.adoc [hunk 13]
```diff
@@ -37,27 +37,35 @@ endif::[]
...
+.Procedure
+. Install Discovery on {ProjectServer}:
++
+ifndef::satellite,orcharhino[]
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} \
+--enable-foreman-plugin-discovery \
+--enable-foreman-proxy-plugin-discovery \
+--foreman-proxy-plugin-discovery-source-url=http://downloads.theforeman.org/discovery/releases/3.5/
```
Comment (maximiliankolb) [inline guides/common/modules/proc_configuring-the-discovery-service.adoc:49]: Should this be "nightly" for "master" branch, and "3.5" for our 3.5 branch?

File: guides/common/modules/proc_installing-the-scc-manager.adoc [hunk 14]
```diff
@@ -1,14 +1,33 @@
 [id="Installing_the_SCC_Manager_Plugin_{context}"]
 = Installing the SCC Manager Plug-in
 
-Perform the following step to install the _SCC Manager_ plug-in on your {Project}.
+Perform the following steps to install the _SCC Manager_ plug-in on your {Project}.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-the-scc-manager.adoc:4]: IMHO something went wrong during rebasing to master. Would you mind having a look at this again? Thanks!

**Review by maximiliankolb — CHANGES_REQUESTED** [review]

File: guides/common/modules/proc_configuring-the-discovery-service.adoc (hunk 13 above)
Comment (evgeni) [inline guides/common/modules/proc_configuring-the-discovery-service.adoc:49]:
```suggestion
--foreman-proxy-plugin-discovery-install-images=true
```

You need install images, but not the URL, you removed the wrong line ;)

Comment (evgeni) [reply guides/common/modules/proc_configuring-the-discovery-service.adoc:49]: nope, that line just shouldn't be there ;)
see https://github.com/theforeman/foreman-documentation/pull/1773/files#r1023674962

File: guides/common/modules/proc_configuring-the-discovery-service.adoc [hunk 15]
```diff
@@ -37,27 +37,35 @@ endif::[]
...
-You can build a PXE boot image from this ISO using the `livecd-iso-to-pxeboot` tool.
-The tool saves this PXE boot image in the `/var/lib/tftpboot/boot` directory.
-For more information, see xref:Building_a_Discovery_Image_{context}[].
+ifdef::satellite,orcharhino[]
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} \
+--enable-foreman-plugin-discovery \
+--enable-foreman-proxy-plugin-discovery \
+--foreman-proxy-plugin-discovery-install-images=false
```
Comment (evgeni) [inline guides/common/modules/proc_configuring-the-discovery-service.adoc:58]: I'd argue this is the default and doesn't need to be mentioned?

File: guides/common/modules/proc_installing-the-discovery-service.adoc [hunk 16]
```diff
@@ -9,33 +9,29 @@ Complete the following procedure to enable the Discovery service on {SmartProxyS
...
 ----
 endif::[]
-ifdef::foreman-el,katello[]
+ifndef::satellite,orcharhino[]
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # {foreman-installer} \
 --enable-foreman-proxy-plugin-discovery \
---foreman-proxy-plugin-discovery-install-images=true \
 --foreman-proxy-plugin-discovery-source-url=http://downloads.theforeman.org/discovery/releases/3.5/
```
Comment (evgeni) [inline guides/common/modules/proc_installing-the-discovery-service.adoc:22]: Please keep `--foreman-proxy-plugin-discovery-install-images=true` and remove `--foreman-proxy-plugin-discovery-source-url=http://downloads.theforeman.org/discovery/releases/3.5/`

File: guides/common/modules/proc_installing-the-scc-manager.adoc [hunk 17]
```diff
@@ -1,14 +1,33 @@
...
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {foreman-installer} --enable-foreman-plugin-scc-manager
+# ssh root@{foreman-example-com}
+----
+. Install the SCC Manager plug-in on your {Project}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {package-install-project} -y rubygem-foreman_scc_manager
```
Comment (evgeni) [inline guides/common/modules/proc_installing-the-scc-manager.adoc:17]: Seems the rebase on https://github.com/theforeman/foreman-documentation/pull/1800 went wrong ;)

Comment (mjivraja) [reply guides/common/modules/proc_installing-the-scc-manager.adoc:17]: I rebased it and went with Ewoud's change. I hope, it is resolved. :crossed_fingers:

File: guides/common/modules/proc_installing-the-discovery-service.adoc (hunk 16 above)
Comment (mjivraja) [reply guides/common/modules/proc_installing-the-discovery-service.adoc]: Addressed :+1:

File: guides/common/modules/proc_configuring-the-discovery-service.adoc (hunk 15 above)
Comment (mjivraja) [reply guides/common/modules/proc_configuring-the-discovery-service.adoc]: Addressed :+1:

File: guides/common/modules/proc_configuring-the-discovery-service.adoc (hunk 13 above)
Comment (mjivraja) [reply guides/common/modules/proc_configuring-the-discovery-service.adoc]: Addressed :+1:

File: guides/common/modules/proc_installing-the-scc-manager.adoc (hunk 14 above)
Comment (mjivraja) [reply guides/common/modules/proc_installing-the-scc-manager.adoc:4]: I rebased it. I hope, there is no more issue :crossed_fingers:

File: guides/common/modules/proc_configuring-the-discovery-service.adoc (hunk 13 above)
Comment (mjivraja) [reply guides/common/modules/proc_configuring-the-discovery-service.adoc]: Addressed :+1:

**Comment by ekohl** [comment]
> A build failed. Probably `linkchecker` ?

Yes. It looks like without `linkchecker-tryer` it's a bit less reliable.

File: guides/common/modules/proc_installing-the-scc-manager.adoc (hunk 17 above)
Comment (ekohl) [reply guides/common/modules/proc_installing-the-scc-manager.adoc:17]: Doesn't look like it. You reverted all the changes.

Comment (mjivraja) [reply guides/common/modules/proc_installing-the-scc-manager.adoc:17]: Oww gosh, I accepted the incoming changes. :face_with_spiral_eyes:

File: guides/common/modules/proc_configuring-the-discovery-service.adoc [hunk 18]
```diff
@@ -37,27 +37,34 @@ endif::[]
...
-The `foreman-discovery-image` package installs the Discovery ISO to the `/usr/share/foreman-discovery-image/` directory.
-You can build a PXE boot image from this ISO using the `livecd-iso-to-pxeboot` tool.
-The tool saves this PXE boot image in the `/var/lib/tftpboot/boot` directory.
-For more information, see xref:Building_a_Discovery_Image_{context}[].
+ifdef::satellite,orcharhino[]
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} \
+--enable-foreman-plugin-discovery \
+--enable-foreman-proxy-plugin-discovery \
```
Comment (evgeni) [inline guides/common/modules/proc_configuring-the-discovery-service.adoc:57]:
```suggestion
--enable-foreman-proxy-plugin-discovery
```

Comment (mjivraja) [reply guides/common/modules/proc_configuring-the-discovery-service.adoc]: Addressed :+1:

File: guides/common/modules/proc_installing-the-discovery-service.adoc [hunk 19]
```diff
@@ -9,33 +9,28 @@ Complete the following procedure to enable the Discovery service on {SmartProxyS
 ifdef::satellite,orcharhino[]
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {package-install-project} foreman-discovery-image tfm-rubygem-smart_proxy_discovery
+# {foreman-installer} \
+--enable-foreman-proxy-plugin-discovery \
```
[the remark this answers is not in the document]
Comment (mjivraja) [reply guides/common/modules/proc_installing-the-discovery-service.adoc]: Addressed :+1:

File: guides/common/modules/proc_installing-the-scc-manager.adoc (hunk 17 above)
Comment (mjivraja) [reply guides/common/modules/proc_installing-the-scc-manager.adoc]: Fixed :+1:

File: guides/common/modules/proc_installing-the-scc-manager.adoc (hunk 14 above)
Comment (mjivraja) [reply guides/common/modules/proc_installing-the-scc-manager.adoc]: Fixed :+1:

File: guides/common/modules/proc_configuring-the-discovery-service.adoc [hunk 20]
```diff
@@ -37,27 +37,35 @@ endif::[]
...
 endif::[]
-* `tfm-rubygem-smart_proxy_discovery`
-
-The `tfm-rubygem-foreman_discovery` package contains the {Project} plug-in to handle discovered nodes, connections, and necessary database structures, and API.
-
-ifdef::satellite[]
-The `foreman-discovery-image` package installs the Discovery ISO to the `/usr/share/foreman-discovery-image/` directory.
-You can build a PXE boot image from this ISO using the `livecd-iso-to-pxeboot` tool.
-The tool saves this PXE boot image in the `/var/lib/tftpboot/boot` directory.
-For more information, see xref:Building_a_Discovery_Image_{context}[].
```
[the remark this answers is not in the document]
Comment (mjivraja) [reply guides/common/modules/proc_configuring-the-discovery-service.adoc]: Addressed :+1:

File: guides/common/modules/proc_configuring-the-discovery-service.adoc [hunk 21]
```diff
@@ -37,27 +37,35 @@ endif::[]
...
+----
+{package-install-project} foreman-discovery-image
+----
 endif::[]
-
-The `tfm-rubygem-smart_proxy_discovery` package configures {SmartProxyServer}, such as the integrated {SmartProxy} of {ProjectServer}, to act as a proxy for the Discovery service.
-
 ifndef::satellite[]
 You can use the `{foreman-installer}` tool to install the packages and the Discovery image.
 endif::[]
```
[the remark this answers is not in the document]
Comment (mjivraja) [reply guides/common/modules/proc_configuring-the-discovery-service.adoc]: Removed :+1:

**Comment by mjivraja** [comment]
Can I assume `linkchecker` again?

File: guides/common/modules/proc_configuring-the-discovery-service.adoc (hunk 13 above)
Comment (ekohl) [reply guides/common/modules/proc_configuring-the-discovery-service.adoc:49]: Still not addressed. Please carefully review your changes before saying it is addressed.

**Review by ekohl — COMMENTED** [review]
Perhaps this is just bugging me, but if you look at both files where there are instructions for Satellite and non-Satellite. In one file you have S first, then NS while in the other file you have NS and then S. That makes it harder to review.

Note that the only difference in the last half of the document is the `--enable-foreman-plugin-discovery` option that is passed on `{ProjectServer}` but isn't passed on `{SmartProxyServer}`. Other than that the procedure is supposed to be the same commands.

File: guides/common/modules/proc_configuring-the-discovery-service.adoc [hunk 22]
```diff
@@ -37,27 +37,41 @@ endif::[]
...
-* `foreman-discovery-image`
-endif::[]
-* `tfm-rubygem-smart_proxy_discovery`
-
-The `tfm-rubygem-foreman_discovery` package contains the {Project} plug-in to handle discovered nodes, connections, and necessary database structures, and API.
-
+.Procedure
+. Install Discovery on {ProjectServer}:
++
 ifdef::satellite[]
```
Comment (ekohl) [inline guides/common/modules/proc_configuring-the-discovery-service.adoc:43]: @maximiliankolb is this bit of text something you'd also want for Orcharhino?

File: guides/common/modules/proc_configuring-the-discovery-service.adoc [hunk 23]
```diff
@@ -37,27 +37,41 @@ endif::[]
...
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} \
+--enable-foreman-plugin-discovery \
+--enable-foreman-proxy-plugin-discovery
+--foreman-proxy-plugin-discovery-install-images=true
+----
+endif::[]
++
+. Install `foreman-discovery-image`:
```
Comment (ekohl) [inline guides/common/modules/proc_configuring-the-discovery-service.adoc:69]: Please move this into the `ifdef::satellite,orcharhino[]` block above since logically it's the same block.

File: guides/common/modules/proc_configuring-the-discovery-service.adoc [hunk 24]
```diff
@@ -37,27 +37,41 @@ endif::[]
...
-
-ifndef::satellite[]
-You can use the `{foreman-installer}` tool to install the packages and the Discovery image.
++
+ifdef::satellite,orcharhino[]
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} \
+--enable-foreman-plugin-discovery \
+--enable-foreman-proxy-plugin-discovery
```
Comment (ekohl) [inline guides/common/modules/proc_configuring-the-discovery-service.adoc:55]: You're missing a backslash here and also below.
```suggestion
--enable-foreman-proxy-plugin-discovery \
```

**Review by ekohl — CHANGES_REQUESTED** [review]
It looks like the `ifdefs` are a bit messed up.

File: guides/common/modules/proc_configuring-the-discovery-service.adoc (hunk 24 above)
Comment (mjivraja) [reply guides/common/modules/proc_configuring-the-discovery-service.adoc:55]: I removed it from Evgeni's [suggestion](https://github.com/theforeman/foreman-documentation/pull/1773#discussion_r1024033117)

File: guides/common/modules/proc_configuring-the-discovery-service.adoc (hunk 23 above)
Comment (mjivraja) [reply guides/common/modules/proc_configuring-the-discovery-service.adoc]: Addressed :+1:

File: guides/common/modules/proc_configuring-the-discovery-service.adoc (hunk 22 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_configuring-the-discovery-service.adoc:42]: Yes. Please make it "if satellite or orcharhino"

Comment (mjivraja) [reply guides/common/modules/proc_configuring-the-discovery-service.adoc]: Addressed :+1:

File: guides/common/modules/proc_configuring-the-discovery-service.adoc (hunk 24 above)
Comment (ekohl) [reply guides/common/modules/proc_configuring-the-discovery-service.adoc:53]: The current instruction is invalid because what you now wrote is actually 2 commands. First you suggest:
```
{foreman-installer} \
--enable-foreman-plugin-discovery \
--enable-foreman-proxy-plugin-discovery
```

And then
```
--foreman-proxy-plugin-discovery-install-images=false
```

This is because the shell interprets every line as a command, except when it ends with a `\`. Then it's seen as a continuation.

@evgeni suggested to drop `--foreman-proxy-plugin-discovery-install-images=false` (https://github.com/theforeman/foreman-documentation/pull/1773/files#r1023676046) and that's one way. Then you should remove the `\`. You now added back the argument, so it must be there.

So you should either drop the `...-images=...` argument (the best option) or add the backslash, but the current instruction must be changed.

Comment (mjivraja) [reply guides/common/modules/proc_configuring-the-discovery-service.adoc:53]: Agree and Addressed :+1:

**Review by ekohl — APPROVED** [review]
This looks correct to me. @maximiliankolb @evgeni any comments before merging?

File: guides/common/modules/proc_installing-the-discovery-service.adoc [hunk 25]
```diff
@@ -9,33 +9,29 @@ Complete the following procedure to enable the Discovery service on {SmartProxyS
...
---foreman-proxy-plugin-discovery-source-url=http://downloads.theforeman.org/discovery/releases/3.5/
+--foreman-proxy-plugin-discovery-install-images=true
 ----
 endif::[]
-. Restart {Project} services:
-+
+ifdef::satellite,orcharhino[]
+. Install `foreman-discovery-image`:
 [options="nowrap" subs="+quotes,attributes"]
++
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-the-discovery-service.adoc:27]:
```suggestion
+
[options="nowrap" subs="+quotes,attributes"]
```

**Review by maximiliankolb — CHANGES_REQUESTED** [review]
One small comment (please order the lines), rest LGTM!

File: guides/common/modules/proc_installing-the-discovery-service.adoc (hunk 25 above)
Comment (mjivraja) [reply guides/common/modules/proc_installing-the-discovery-service.adoc]: Addressed :+1: 
Also covered at `proc_configuring-the-discovery-service.adoc`

**Comment by maximiliankolb** [comment]
Change looks good to me. I've restarted GHA; let's see if it's green.

**Comment by maximiliankolb** [comment]
Please rebase to master and squash your commits. If you enable autosquash, git will do this automatically for you. Either use `--autosquash` or extend your `.gitconfig` file:

`````
[rebase]
        autosquash = true
````

**Comment by ekohl** [comment]
> Please rebase to master and squash your commits. If you enable autosquash, git will do this automatically for you. Either use `--autosquash` or extend your `.gitconfig` file:

TIL you can actually set it as a default. Note you can also use `git config --global rebase.autosquash true` to set this.

**Comment by mjivraja** [comment]
> Please rebase to master and squash your commits. If you enable autosquash, git will do this automatically for you. Either use `--autosquash` or extend your `.gitconfig` file:
> 
> ```
> [rebase]
>         autosquash = true
> ```

Thanks @maximiliankolb and @ekohl I will be playing with `.gitconfig` for sure.

**Review by maximiliankolb — APPROVED** [review]
Thanks @mjivraja  LGTM!

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked: ✅ 
   b7aedc949..f999e9e69  3.5 -> 3.5
   022925f27..e78eccd13  3.4 -> 3.4
   bbe577424..c6a75e4d2  3.3 -> 3.3
